src/ext/SandboxFeature.tsx

```diff
@@ -74,4 +75,7 @@ export default memo(SandboxFeature) as typeof SandboxFeature<
   any
 >;
 
-export const TestSandboxFeature = memo(SandboxFeature) as typeof SandboxFeature;
+export const TestSandboxFeature = memo(SandboxFeature) as typeof SandboxFeature<
```

Comment:
This type should probably be moved into the test component itself, to avoid any confusion